package.json

```diff
@@ -21,6 +21,10 @@
     "e2e:smoke": "playwright test src/e2e/ --grep @smoke",
     "cron:monthly-activity": "node dist/scripts/cronjobs/monthlyActivity.js",
     "cron:breach-alerts": "node src/scripts/emailBreachAlerts.js",
+    "cron:db-delete-unverified-subscribers": "node scripts/delete-unverified-subscribers.js",
+    "cron:db-pull-breaches": "node src/scripts/syncBreaches.js",
+    "cron:remote-settings-pull-breaches": "node scripts/updatebreaches.js",
```

Comment:
Does this one actually run? It's [referencing](https://github.com/mozilla/blurts-server/blob/7725b368fcf9c04b82ae44686495a109841e52e7/scripts/updatebreaches.js#L10) a file that doesn't exist, and I'm not sure if it's still relevant - does Firefox still show breaches anywhere? I don't see anything in `about:protections` anymore.

Reply:
OK, so from Slack it sounds like it's disabled for now, even though the script exists. I'll leave it in here since the script and the reference to it still exist, so the disabling might not necessarily have been (intended to be) permanent yet.